### PR TITLE
Add schema for document class

### PR DESCRIPTION
**Pull request by iross**
- Added a quick schema from the existing prototype document store.

Sorry about the noise from the README re-ordering that got spit out -- if it gets auto-generated somewhere, I can remove the changes from this PR.

### CODE REVIEW

**Comment by davenquinn** [comment]
@iross looks like this is documents only — is there anything to include around entities (i.e. to hold the COSMOS ingestions)?

**Comment by iross** [comment]
@davenquinn Just pushed the extractions as well

File: README.md [hunk 1]
```diff
@@ -524,30 +532,81 @@ classDiagram
...
+    <summary>document</summary>
+
+```mermaid
+classDiagram
+
+    class Document {
+        id: UUID
+        title: str
+        is_public: bool
+        xdd_id: Optional[str] = None
```
Comment (jgawrilo) [inline README.md:569]: @iross I don't think we want the CDR to be tightly coupled to XDD - suggest removing the xdd_id.

**Comment by jgawrilo** [comment]
@iross @davenquinn I think we want the CDR to be agnostic of knowing about XDD.

**Comment by davenquinn** [comment]
@jgawrilo @iross doesn't the fact that the field is optional mean that the CDR is agnostic about it? It makes sense that we'd want the two systems to be decoupled, but recording which documents are also found in external systems (incl/especially xDD) seems important.

Maybe this could be generalized into a "provenance" model, e.g.,
`document_source`, `document_source_id`, `document_source_url`
or something like that?

**Comment by iross** [comment]
I think @davenquinn has a good point here, and not just on the xDD side (as near and dear to my heart as it is 😉). This is a pretty minimal initial schema we implemented, and it lacks things that might be desirable/necessary to track. For example, the NI 43-101 mining docs all have IDs/URIs in both the GeoKB and Zotero origins that would also fit into a provenance model.

**Comment by jgawrilo** [comment]
> @jgawrilo @iross doesn't the fact that the field is optional mean that the CDR is agnostic about it? It makes sense that we'd want the two systems to be decoupled, but recording which documents are also found in external systems (incl/especially xDD) seems important.
> 
> Maybe this could be generalized into a "provenance" model, e.g., `document_source`, `document_source_id`, `document_source_url` or something like that?

Please see https://github.com/DARPA-CRITICALMAAS/cdr_schemas/pull/20 as our proposal

**Comment by jgawrilo** [comment]
Our proposal based on review of this PR: https://github.com/DARPA-CRITICALMAAS/cdr_schemas/pull/20

**Pull request closed without merging**